### PR TITLE
[WM-1564] Submission Config page

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "qs": "^6.10.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-dropzone": "^14.2.3",
```
Comment (cjllanwarne) [inline package.json:27]: I thought dropzones ended up being unneeded?

File: src/pages/SubmissionConfig.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import { ButtonPrimary, headerBar, Link, Select } from 'src/components/common'
+import StepButtons from 'src/components/StepButtons'
+import { Ajax } from 'src/libs/ajax'
+import { notify } from 'src/libs/notifications'
+import { useCancellation, useOnMount } from 'src/libs/react-utils'
+import * as Utils from 'src/libs/utils'
+
+
+const methodId = '64b5bc5e-85cf-4aff-b522-01471b88b950' // TODO: get this from the previous page
+// const wdlUrl = 'https://raw.githubusercontent.com/broadinstitute/viral-pipelines/master/pipes/WDL/workflows/fastq_to_ubam.wdl'
```
Comment (cjllanwarne) [inline src/pages/SubmissionConfig.js:13]: If unneeded, I would remove this

**Review by cjllanwarne — APPROVED** [review]
LGTM, a few TOL comments

**Review by kpierre13 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mspector) [reply package.json:27]: you are correct! I'll remove

**Pull request merged**